code/datums/loadout/item_representation/_item_representation.dm

```diff
@@ -27,6 +27,9 @@
 /datum/item_representation/proc/instantiate_object(datum/loadout_seller/seller, master = null, mob/living/user)
 	if(seller && !bypass_vendor_check && !buy_item_in_vendor(item_type, seller, user))
 		return
+	if(!text2path(item_type))
+		to_chat(user, span_warning("[item_type] is not a correct path, it as probably changed or removed."))
```

Comment:
```suggestion
		to_chat(user, span_warning("[item_type] in your loadout is an invalid path, it as probably changed or removed."))
```
Also shouldnt it remove it from the loadout so it doesnt happen again

Reply:
See comment

Reply:
way too much work for something that will rarely happen, and that you can fix by saving your loadout another time

Reply:
not as simple as a -= 

an incorrect item can be saved on anything. In a storage, on an armor, can be a gun attachment, etc.